crates/bevy_pbr/src/deferred/deferred_lighting.wgsl

```diff
@@ -64,7 +65,15 @@ fn fragment(in: FullscreenVertexOutput) -> @location(0) vec4<f32> {
 #ifdef SCREEN_SPACE_AMBIENT_OCCLUSION
         let ssao = textureLoad(screen_space_ambient_occlusion_texture, vec2<i32>(in.position.xy), 0i).r;
         let ssao_multibounce = gtao_multibounce(ssao, pbr_input.material.base_color.rgb);
-        pbr_input.occlusion = min(pbr_input.occlusion, ssao_multibounce);
+        pbr_input.diffuse_occlusion = min(pbr_input.diffuse_occlusion, ssao_multibounce);
+
+        // Neubelt and Pettineo 2013, "Crafting a Next-gen Material Pipeline for The Order: 1886"
+        let NdotV = max(dot(pbr_input.N, pbr_input.V), 0.0001); 
+        var perceptual_roughness: f32 = pbr_input.material.perceptual_roughness;
+        let roughness = lighting::perceptualRoughnessToRoughness(perceptual_roughness);
+
+        // Use ssao to estimate the specular occlusion. [Lagarde et al., 2014]
+        pbr_input.specular_occlusion =  clamp(pow(NdotV + ssao, exp2(-16.0 * roughness - 1.0)) - 1.0 + ssao, 0.0, 1.0);
```

Comment:
Does it make sense to use only the SSAO here, and not min(ssao, baked_ao)? No idea which is better.

For context, the baked AO should be treated kinda like a normal map. It's _not_ for occlusion between multiple meshes (that's what lightmaps are for). It's instead occlusion baked as part of the material, such as micro-cracks and indents in the material.

We should update the docs to make this a bit more explicit imo.

Reply:
The baked AO is a vec3, I'm not sure how to convert that to a visibility value, maybe luminance?

Reply:
The baked occlusion is a single f32. Check where the texture is read in PBR fragment, we only read the R channel. It's simply cast to a vec3 for compatibility with SSAO multibounce.

Reply:
Thanks, I missed that.

Reply:
We attempted this and decided to revert.